pkg/safeguards/preprocessing/preprocessing_test.go

```diff
@@ -181,33 +181,72 @@ func TestRenderKustomizeManifest_Valid(t *testing.T) {
 	assert.Nil(t, err)
 }
 
+// TODO: later update these tests to validate the file content returned in the ManifestFile struct
+func TestGetManifestFiles(t *testing.T) {
```

Comment:
This looks to be the same kind test as in validate.go? You can combine these 2

Reply:
I will move these to validate_test.go TestRunValidate() and test GetManifestResults() for helm and kustomize as well